setup.py

```diff
@@ -1,6 +1,20 @@
 import io
+import pkgutil
 from setuptools import setup, find_packages
 
+ENTRY_POINTS = {
+    "console_scripts": [
+        "dash-generate-components = "
+        "dash.development.component_generator:cli",
+        "renderer = dash.development.build_process:renderer",
+    ]
+}
+
+# this is not a complete guess, but picking two typical dash dependencies in
+# require-testing.txt
+if pkgutil.find_loader("waitress") and pkgutil.find_loader("percy"):
+    ENTRY_POINTS["pytest11"] = ["dash = dash.testing.plugin"]
```

Comment:
Interesting... but will these exist already if we're installing `dash[testing]` for the first time? Is there no way to directly determine whether the `testing` extra was requested?

Reply:
I tried with a clear env, it we install with `dash[testing]` it adds the entry_point as expected. in the case of only `pip install dash`, no pytest-dash is registered. 

I don't think there is an easy way to directly determine if the `testing` extras is used:
- if we install with editable mode, in `dash.egg-info/PKG-INFO`, we can get information like this. but this will depends on how the package is installed. the simple `pip install dash[testing]` does not contain such information. not sure what happens with `conda install`
```
Provides-Extra: dev
Provides-Extra: testing
```